### ISSUE TITLE
bioc-devel mailing list check still not always accurate

### ISSUE THREAD

**Issue by lshep** [Bioconductor/BiocCheck#205]
I  can confirm the single package builder is using version 1.39.10 
See mailing list https://stat.ethz.ch/pipermail/bioc-devel/2024-January/020119.html 
for email = "richardheery@gmail.com". 

I can confirm the email is registered on bioc-devel but it is report not. 
http://staging.bioconductor.org:8000/job/19968/

**Comment by lshep** [comment]
also someone else this morning: https://github.com/Bioconductor/Contributions/issues/3076#issuecomment-1881357053

**Comment by LiNk-NY** [comment]
I ran this locally and I didn't get the error: 
```r
> checkForBiocDevelSubscription("~/bioc/methodical/")
    Maintainer is subscribed to bioc-devel.
> grep("Version:", readLines("~/bioc/methodical/DESCRIPTION"), fixed = TRUE, value = TRUE)
[1] "Version: 0.99.20"
```

and for `mobileRNA`

```r
> checkForBiocDevelSubscription("~/bioc/mobileRNA/")
    Maintainer is subscribed to bioc-devel.
```

**Comment by lshep** [comment]
Well they keep coming in. Here is another one that was passing but now is not:  
https://github.com/Bioconductor/Contributions/issues/3251#issuecomment-1882506846

**Comment by LiNk-NY** [comment]
Thanks, it looks like the bug was only happening with `Maintainer` fields in the tarball. It should be resolved in 979e0d8c3a554243d10e88520ac9ed622a8993df
